src/java.base/share/classes/java/util/Collection.java

```diff
@@ -58,7 +58,7 @@
  * constructors) but all of the general-purpose {@code Collection}
  * implementations in the Java platform libraries comply.
  *
- * <p>Certain methods are specified to be
+ * <p><a id="optional-operations"></a>Certain methods are specified to be
```

Comment:
```suggestion
 * <p id="optional-operations">Certain methods are specified to be
```

Reply:
I want to keep the style as the existing one used in other places in the class.